src/test/java/teammates/test/cases/browsertests/InstructorFeedbackPageUiTest.java

```diff
@@ -250,9 +250,9 @@ private void testAddAction() throws Exception {
         feedbackPage.clickNeverVisibleTimeButton();
 
         //verify that timeFrameTable, instructions and ResponseVisTable are all hidden
-        feedbackPage.verifyHidden(By.id("timeFramePanel"));
-        feedbackPage.verifyHidden(By.id("responsesVisibleFromColumn"));
-        feedbackPage.verifyHidden(By.id("instructionsRow"));
+        assertTrue(feedbackPage.verifyHidden(By.id("timeFramePanel")));
```

Comment:
On a related note, the culprit for this problem may be improperly named method. `verifyHidden` --> `isHidden`? Can be a separate first timer issuer. In fact this issue too could have been a first timer issue :-)